website/utils/url_utils.ts

```diff
@@ -406,6 +440,27 @@ function parseStringRegion(region: string | undefined): PerAxis<[number, number]
   return { x, y, z };
 }
 
+function serializeControlPoints(controlPoints: ControlPoint[]): string {
+  return controlPoints.map((cp) => `${cp.x}:${cp.opacity}:${colorArrayToHex(cp.color)}`).join(",");
+}
+
+function parseControlPoints(controlPoints: string | undefined): ControlPoint[] | undefined {
+  if (!controlPoints || !CONTROL_POINTS_REGEX.test(controlPoints)) {
+    return undefined;
+  }
+  const newControlPoints = controlPoints.split(",").map((cp) => {
+    const [x, opacity, color] = cp.split(":");
+    return {
+      // TODO: Is there an expected range of values for x?
+      x: parseStringFloat(x, -Infinity, Infinity) ?? 0,
```

Comment:
0-255 is "standard," but unfortunately most anything is technically possible after a remap. This is probably as close to correct as you can get